Client/Frontend/Browser/TabManagement/Tab+ChangeUserAgent.swift

```diff
@@ -13,7 +13,8 @@ extension Tab {
         }()
 
         private static var baseDomainList: Set<String> = {
-            if let hosts = NSKeyedUnarchiver.unarchiveObject(withFile: ChangeUserAgent.file.path) as? Set<String> {
+            if let data = try? Data(contentsOf: ChangeUserAgent.file),
+               let hosts = try? NSKeyedUnarchiver.unarchivedObject(ofClasses: [NSSet.self, NSArray.self, NSString.self], from: data) as? Set<String> {
```

Comment:
Oh those changes are from the other task, they shouldn't be on this branch 🤔

Reply:
damn, I forgot to switch to main before creating this branch, should I try to rebase the branch to main or delete it and recreate the changes from main?

Reply:
As you wish! You can try rebasing, worst case you can close this PR and open a new one by cherry picking the commits for this work